app/src/main/java/net/gsantner/markor/activity/DocumentEditAndViewFragment.java

```diff
@@ -94,6 +96,7 @@ public static DocumentEditAndViewFragment newInstance(final @NonNull Document do
 
     private HighlightingEditor _hlEditor;
     private WebView _webView;
+    private ViewStub _webViewStub;
```

Comment:
The webview is the heaviest view in the document fragment. This loads it lazily when necessary